packages/gluestick-generators/src/requireGenerator.js

```diff
@@ -8,14 +8,17 @@ const { convertToCamelCase, convertToKebabCase } = require('./utils');
 // $FlowFixMe
 module.createTemplate = createTemplate;
 
-const PREDEFINED: string = 'predefined';
+const PATH_TO_GLUESTICK_TEMPLATES = '../../gluestick/generator/predefined';
```

Comment:
I'm not sure about this solution, in fact it results in `Cannot find module 'gluestick'`.